File: JAVA/Civitas.java
public class Civitas extends Human {
    // attribute
    private String asalUniversitas, emailEdu;

    // constructor
    public Civitas() {
        this.asalUniversitas = "";
        this.emailEdu = "";
    } 

    public Civitas(String asalUniversitas, String emailEdu) {
        this.asalUniversitas = asalUniversitas;
        this.emailEdu = emailEdu;
    }

    // getter and setter
    public String getAsalUniversitas() {
        return this.asalUniversitas;
    }

    public String getEmailEdu() {
        return this.emailEdu;
    }

    public void setAsalUniversitas(String asalUniversitas) {
        this.asalUniversitas = asalUniversitas;
    }

    public void setEmailEdu(String emailEdu) {
        this.emailEdu = emailEdu;
    }

    // method
    public void absen() {
        System.out.println("bang sebut nama");
    }
}

File: PHP/Civitas.php
<?php

require('Human.php');

class Civitas extends Human {
    // attribute
    private $asalUniversitas, $emailEdu;

    // constructor
    public function __construct() {
        $this->asalUniversitas = "";
        $this->emailEdu = "";
    }

    // getter and setter
    public function getAsalUniversitas() {
        return $this->asalUniversitas;
    }

    public function getEmailEdu() {
        return $this->emailEdu;
    }

    public function setAsalUniversitas($asalUniversitas) {
        $this->asalUniversitas = $asalUniversitas;
    }

    public function setEmailEdu($emailEdu) {
        $this->emailEdu = $emailEdu;
    }

    // method
    public function absen() {
        echo "bang sebut nama" . '<br>';
    }
}


?>